samples/SelfHostServer/Startup.cs

```diff
@@ -37,6 +37,7 @@ public static void Main(string[] args)
                 .UseStartup<Startup>()
                 .UseHttpSys(options =>
                 {
+                    options.Listen("http://localhost:5000");
```

Comment:
To illustrate how to specify a specific address using the direct configuration api.